src/main/java/com/termos/dto/BookDTO.java

```diff
@@ -0,0 +1,31 @@
+package com.termos.dto;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.sql.Date;
+
+public class BookDTO extends AbstractDTO {
+    @NotNull
+    @NotEmpty
+    public String bookId;
+
+    @NotNull
+    @NotEmpty
+    public String title;
+
+    @NotNull
+    @NotEmpty
+    public String author;
+
+    @NotNull
+    @NotEmpty
+    public float price = 0.00f;
```

Comment:
NotNull i NotEmpty nie ma tutaj sensu ponieważ float jest typem prostym (nie jest to obiekt). Taki typ nie może być pusty i zawsze ma jakąś wartość